src/app/map/[id]/components/controls/layers/MarkersControl/MarkersList.tsx

```diff
@@ -128,24 +132,24 @@ export default function MarkersList() {
           ? getNewLastPosition(folderMarkers)
           : getNewFirstPosition(folderMarkers);
 
-        preparePlacedMarkerUpdate({
+        updatePlacedMarker({
```

Comment:
note: we lose a tiny bit of performance here. The behaviour before was to only send a request once the user was done dragging the marker, but now it will do a request while the user is still dragging, but crossing over into different folders (if I understand the code correctly). This is probably fine, though, and makes for simpler code.